graph/simple/simple.go

```diff
@@ -18,6 +18,10 @@ func (n Node) ID() int64 {
 	return int64(n)
 }
 
+func newSimpleNode(id int) graph.Node {
```

Comment:
Why use this instead of simple calling `Node(id)`?

Reply:
Because the `iterator.NewImplicitNodes` function takes a `func(int) graph.Node` and `graph.Node()` is not a function.

Reply:
Oh, I see. Thanks for clarifying.